frontend/schemaCMS/src/shared/components/tabs/tabs.styles.js

```diff
@@ -26,9 +36,14 @@ export const Tab = styled.div`
   }
 `;
 
-export const TabContent = styled(Link)`
+export const TabContentLink = styled(Link)`
```

Comment:
You can common styles put in `css` for example:
```
const tabContentStyles = css`
  color: ${({ theme: { text } }) => text};
  padding: 16px;
```

and later 
```
export const TabContentButton = styled.div`
  ${tabContentStyles};
`;
```

Reply:
Thanks for the tip Rafał :)